packages/launchpad/src/setup/ManualInstall.vue

```diff
@@ -70,19 +71,4 @@ const props = defineProps<{
   gql: ManualInstallFragment
   packagesInstalled: string[]
 }>()
-
-const commands = {
-  'npm': 'npm install -D ',
-  'pnpm': 'pnpm install -D ',
-  'yarn': 'yarn add -D ',
-}
-
-const installDependenciesCode = computed(
```

Comment:
Banished to the data-context where it belongs https://github.com/cypress-io/cypress/pull/20306/files#diff-2f8e799bd85298d61d6987bdde47d59cea5cd14a13ebc145c9fe4e79f44553e1R50